controllers/healthcheck_controller.go

```diff
@@ -395,6 +398,7 @@ func (r *HealthCheckReconciler) createSubmitWorkflow(ctx context.Context, log lo
 	workflow.SetGroupVersionKind(wfGvk)
 	workflow.SetNamespace(hc.Spec.Workflow.Resource.Namespace)
 	workflow.SetGenerateName(hc.Spec.Workflow.GenerateName)
+	workflow.SetLabels(r.workflowLabels)
```

Comment:
this should also be added to remedy workflow.

Reply:
added to remedy workflow.